src/lib/components/collections/req-res-section/sub-components/sub-components-header/CodeMirrorInput.svelte

```diff
@@ -168,6 +168,37 @@
     );
   };
 
+  const wordHover = hoverTooltip((view, pos, side) => {
```

Comment:
Have we repeated this code?

Reply:
Yeah we have used hoverTooltip also but the use case was different like we were using it to check and color envs, I tried to add this use case in that code itself but that then it was not working as expected.